tests/test_call_policies.py

```diff
@@ -69,6 +69,35 @@ def test_keep_alive_return_value(capture):
     """
 
 
+def test_keep_alive_single():
+    """Issue #1251 - patients are stored multiple times when given to the same nurse"""
+
+    nurse, p1, p2 = UserType(), UserType(), UserType()
```

Comment:
Same as #1253 - I would like to modify this test to make `refcount()` Pythonic, and check behavior after `nurse` is GC'd.

Reply:
Done. Cherry-picked 3fb4858be7afe6ed75e22c0026c480c3ea5f55f9 from #1253.